apps/nowcasting-app/constant.ts

```diff
@@ -3,7 +3,7 @@ export const API_PREFIX = process.env.NEXT_PUBLIC_API_PREFIX || "https://api-dev
 export const SITES_API_PREFIX =
   process.env.NEXT_PUBLIC_SITES_API_PREFIX || "https://api-site-dev.quartz.solar";
 export const MAX_POWER_GENERATED = 500;
-export const MAX_NATIONAL_GENERATION_MW = 12000;
+export const MAX_NATIONAL_GENERATION_MW = 18000;
```

Comment:
If we have the additional logic you've added for cases where we do go over this threshold, I think we can maybe keep the lower max value here so that the overall graph can still use a good amount of the height, and avoid startling the user into thinking our generation is (visually) much less than it was before – could we try it with the `12000` back in and see if it works for both below and above that (UK solar is very rarely going to be hitting these heights for the near future!)